### PR TITLE
Fix styling for commiter on diff view

### PR DESCRIPTION
**Pull request by CirnoT**
I've tried adjusting width here already in https://github.com/go-gitea/gitea/pull/11441 but it turns out that it doesn't look so well when there are multiple parents (see image on https://github.com/go-gitea/gitea/issues/11709)

chrome_2020-05-31_23-45-46
chrome_2020-05-31_23-52-02

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1650,6 +1650,7 @@ diff.review.placeholder = Review comment
 diff.review.comment = Comment
 diff.review.approve = Approve
 diff.review.reject = Request changes
+diff.commited_by = commited by
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]: typo: `committed`

**Comment by silverwind** [comment]
I'd probably remove the commit icon and do:

User1 authored 25 minutes ago
User2 committed 25 minutes ago

**Comment by CirnoT** [comment]
The whole point of splitting the message is to make is shorter, for long emails or names the message easily goes into second line and I'd like to avoid it as it looks weird when only part of '2 days ago' is in a second line.

**Comment by CirnoT** [comment]
Also duplicating time on committed user looks weird as hell

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (CirnoT) [reply options/locale/locale_en-US.ini]:
```suggestion
diff.commited_by = committed by
```

**Comment by silverwind** [comment]
Right, date duplication is weird. Maybe follow what GitHub does.

**Comment by CirnoT** [comment]
That would make message even longer

**Comment by CirnoT** [comment]
GitHub can do it for two reasons:
- They don't show user email at all, we do.
- They can stack away avatars nicely, we can't.

**Comment by silverwind** [comment]
If space is scarce, maybe we should move the e-mail to a `title` attribute but I guess such a change needs to be done on more places.

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -1650,6 +1650,7 @@ diff.review.placeholder = Review comment
 diff.review.comment = Comment
 diff.review.approve = Approve
 diff.review.reject = Request changes
+diff.commited_by = committed by
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]:
```suggestion
diff.committed_by = committed by
```

**Comment by CirnoT** [comment]
Also given how our default trust model for signing works, I find it useful to have it obvious who committed and who authored.

File: templates/repo/commit_page.tmpl [hunk 3]
```diff
@@ -45,19 +45,22 @@
...
 					{{if or (ne .Commit.Committer.Name .Commit.Author.Name) (ne .Commit.Committer.Email .Commit.Author.Email)}}
-						<span> </span>
-						{{if ne .Verification.CommittingUser.ID 0}}
-							<img class="ui avatar image" src="{{.Verification.CommittingUser.RelAvatarLink}}" />
-							<a href="{{.Verification.CommittingUser.HomeLink}}"><strong>{{.Commit.Committer.Name}}</strong> &lt;{{.Commit.Committer.Email}}&gt;</a>
-						{{else}}
-							<img class="ui avatar image" src="{{AvatarLink .Commit.Committer.Email}}" />
-							<strong>{{.Commit.Committer.Name}}</strong>
-						{{end}}
+						<div class="commited-by">
```
Comment (silverwind) [inline templates/repo/commit_page.tmpl]:
```suggestion
						<div class="committed-by">
```

File: templates/repo/commit_page.tmpl [hunk 4]
```diff
@@ -45,19 +45,22 @@
...
+							<span class="text grey">{{svg "octicon-git-commit" 16}}{{.i18n.Tr "repo.diff.commited_by"}}</span>
```
Comment (silverwind) [inline templates/repo/commit_page.tmpl]:
```suggestion
							<span class="text grey">{{svg "octicon-git-commit" 16}}{{.i18n.Tr "repo.diff.committed_by"}}</span>
```

File: web_src/less/_repository.less [hunk 5]
```diff
@@ -2375,6 +2375,24 @@
             }
         }
     }
+
+    &.diff .commited-by {
```
Comment (silverwind) [inline web_src/less/_repository.less]:
```suggestion
    &.diff .committed-by {
```

**Review by silverwind — APPROVED** [review]
Guess more improvements can be done later.

**Comment by techknowlogick** [comment]
ping LG-TM

**Comment by techknowlogick** [comment]
@CirnoT please send backport :)

**Pull request merged**